include/acl_selectors.php

```diff
@@ -613,9 +626,9 @@ function acl_lookup(App $a, $out_type = 'json') {
 	if ($conv_id) {
 		/* if $conv_id is set, get unknow contacts in thread */
 		/* but first get know contacts url to filter them out */
-		function _contact_link($i){ return dbesc($i['link']); }
-		$known_contacts = array_map(_contact_link, $contacts);
-		$unknow_contacts=array();
+		function _contact_link($i) { return dbesc($i['link']); }
+		$known_contacts = array_map('_contact_link', $contacts);
```

Comment:
Nice, however I would have preferred to see a closure used here instead of a named function.